_pages/en_US/restoring-nand.md

```diff
@@ -12,11 +12,11 @@ Do not skip *anything* on this page, any mistake greatly increases chance of bri
 
 First, a few safer alternatives to why you might want to do this:
 - Installing DSiWare can be done using hiyaCFW or TWiLight Menu++
-- Recovering pictures can be done using ninfs, in combination with hiyaCFW or TWiLight Menu++ if you want them on console
+- Recovering pictures can be done using [ninfs](https://github.com/ihaveamac/ninfs/releases), in combination with hiyaCFW or TWiLight Menu++ if you want them on console
```

Comment:
The hiyaCFW helper also has extracting the photos built in now, maybe mention that